Changelog for the deviceservice

shaaf-c8y opened the pull request.

shaaf-c8y committed:
Add DeviceService connectivity plugin

shaaf-c8y committed:
update description for DeviceService

shaaf-c8y committed:
rename and refine documentation

shaaf-c8y committed:
refine title

shaaf-c8y committed:
clear date and version

BeateRixen commented on content/change-logs/analytics/apama-in-c8y-20251029-DeviceService-bundle.md:
```diff
@@ -0,0 +1,23 @@
+---
+date: 
+title: The DeviceService Bundle
```
```suggestion
title: New DeviceService bundle
```

BeateRixen commented on content/change-logs/analytics/apama-in-c8y-20251029-DeviceService-bundle.md:
```diff
@@ -0,0 +1,23 @@
+title: The DeviceService Bundle
+change_type:
+  - value: change-inv-3bw8e
+    label: Announcement
```
As this is a preview feature, I suggest setting the change type to "Preview" (and change the value accordingly).

shaaf-c8y replied:
Changed type to "Preview" [c5242ee](https://github.com/Cumulocity-IoT/c8y-docs/pull/3881/commits/c5242ee808146e5c0b45cfb9bf3341c4cde7ec87)

shaaf-c8y committed:
Update content/change-logs/analytics/apama-in-c8y-20251029-DeviceService-bundle.md

Co-authored-by: Beate Rixen <90445236+BeateRixen@users.noreply.github.com>

shaaf-c8y committed:
Update change type label for DeviceService bundle to reflect preview

shaaf-c8y committed:
Update link for DeviceService bundle documentation

shaaf-c8y committed:
add version

skom-c8y commented on content/change-logs/analytics/apama-in-c8y-20251029-DeviceService-bundle.md:
```diff
@@ -0,0 +1,23 @@
+version: 26.261.0
+---
+
+A new **DeviceService** bundle is now available, providing capabilities for consuming and publishing device messages via the MQTT Service. This bundle serves as the replacement for the deprecated Cumulocity MQTT Service.
```
You need to be explicit on what you mean by `Cumulocity MQTT Service` ? do you mean MQTT Service bundle in Apama ? 

Cumulocity MQTT Service is not deprecated, its currently in public preview and will be in GA shortly

shaaf-c8y replied:
Updated [6f47059](https://github.com/Cumulocity-IoT/c8y-docs/pull/3881/commits/6f470594ef54ee8be4df11854529d9ff78a3aee0)

shaaf-c8y committed:
Clarify DeviceService bundle replacement details

Updated the description of the DeviceService bundle to clarify its replacement for the deprecated Apama EPL API.

BeateRixen commented on content/change-logs/analytics/apama-in-c8y-20251029-DeviceService-bundle.md:
```diff
@@ -0,0 +1,23 @@
+version: 26.261.0
+---
+
+A new **DeviceService** bundle is now available, providing capabilities for consuming and publishing device messages via the MQTT Service. This bundle serves as the replacement for the deprecated Apama EPL API for the MQTT Service (com.apama.cumulocity.mqttservice).
```
```suggestion
A new **DeviceService** bundle is now available, providing capabilities for consuming and publishing device messages via the MQTT Service. This bundle serves as the replacement for the deprecated Apama EPL API for the MQTT Service (`com.apama.cumulocity.mqttservice`).
```

BeateRixen committed:
Update content/change-logs/analytics/apama-in-c8y-20251029-DeviceService-bundle.md
